File: src/.NET/fix44/SettlementInstructionRequest.cs
namespace QuickFix44
{

  public class SettlementInstructionRequest : Message
  {
    public SettlementInstructionRequest() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("AV"); }

    public SettlementInstructionRequest(
      QuickFix.SettlInstReqID aSettlInstReqID,
      QuickFix.TransactTime aTransactTime )
    : base(MsgType()) {
      set(aSettlInstReqID);
      set(aTransactTime);
    }

    public void set(QuickFix.SettlInstReqID value)
    { setField(value); }
    public QuickFix.SettlInstReqID get(QuickFix.SettlInstReqID  value)
    { getField(value); return value; }
    public QuickFix.SettlInstReqID getSettlInstReqID()
    { QuickFix.SettlInstReqID value = new QuickFix.SettlInstReqID();
      getField(value); return value; }
    public bool isSet(QuickFix.SettlInstReqID field)
    { return isSetField(field); }
    public bool isSetSettlInstReqID()
    { return isSetField(791); }

    public void set(QuickFix.TransactTime value)
    { setField(value); }
    public QuickFix.TransactTime get(QuickFix.TransactTime  value)
    { getField(value); return value; }
    public QuickFix.TransactTime getTransactTime()
    { QuickFix.TransactTime value = new QuickFix.TransactTime();
      getField(value); return value; }
    public bool isSet(QuickFix.TransactTime field)
    { return isSetField(field); }
    public bool isSetTransactTime()
    { return isSetField(60); }

    public void set(QuickFix.NoPartyIDs value)
    { setField(value); }
    public QuickFix.NoPartyIDs get(QuickFix.NoPartyIDs  value)
    { getField(value); return value; }
    public QuickFix.NoPartyIDs getNoPartyIDs()
    { QuickFix.NoPartyIDs value = new QuickFix.NoPartyIDs();
      getField(value); return value; }
    public bool isSet(QuickFix.NoPartyIDs field)
    { return isSetField(field); }
    public bool isSetNoPartyIDs()
    { return isSetField(453); }

    public class NoPartyIDs: QuickFix.Group
    {
    public NoPartyIDs() : base(453,448,message_order ) {}
    static int[] message_order = new int[] {448,447,452,802,0};
      public void set(QuickFix.PartyID value)
      { setField(value); }
      public QuickFix.PartyID get(QuickFix.PartyID  value)
      { getField(value); return value; }
      public QuickFix.PartyID getPartyID()
      { QuickFix.PartyID value = new QuickFix.PartyID();
        getField(value); return value; }
      public bool isSet(QuickFix.PartyID field)
      { return isSetField(field); }
      public bool isSetPartyID()
      { return isSetField(448); }

      public void set(QuickFix.PartyIDSource value)
      { setField(value); }
      public QuickFix.PartyIDSource get(QuickFix.PartyIDSource  value)
      { getField(value); return value; }
      public QuickFix.PartyIDSource getPartyIDSource()
      { QuickFix.PartyIDSource value = new QuickFix.PartyIDSource();
        getField(value); return value; }
      public bool isSet(QuickFix.PartyIDSource field)
      { return isSetField(field); }
      public bool isSetPartyIDSource()
      { return isSetField(447); }

      public void set(QuickFix.PartyRole value)
      { setField(value); }
      public QuickFix.PartyRole get(QuickFix.PartyRole  value)
      { getField(value); return value; }
      public QuickFix.PartyRole getPartyRole()
      { QuickFix.PartyRole value = new QuickFix.PartyRole();
        getField(value); return value; }
      public bool isSet(QuickFix.PartyRole field)
      { return isSetField(field); }
      public bool isSetPartyRole()
      { return isSetField(452); }

      public void set(QuickFix.NoPartySubIDs value)
      { setField(value); }
      public QuickFix.NoPartySubIDs get(QuickFix.NoPartySubIDs  value)
      { getField(value); return value; }
      public QuickFix.NoPartySubIDs getNoPartySubIDs()
      { QuickFix.NoPartySubIDs value = new QuickFix.NoPartySubIDs();
        getField(value); return value; }
      public bool isSet(QuickFix.NoPartySubIDs field)
      { return isSetField(field); }
      public bool isSetNoPartySubIDs()
      { return isSetField(802); }

      public class NoPartySubIDs: QuickFix.Group
      {
      public NoPartySubIDs() : base(802,523,message_order ) {}
      static int[] message_order = new int[] {523,803,0};
        public void set(QuickFix.PartySubID value)
        { setField(value); }
        public QuickFix.PartySubID get(QuickFix.PartySubID  value)
        { getField(value); return value; }
        public QuickFix.PartySubID getPartySubID()
        { QuickFix.PartySubID value = new QuickFix.PartySubID();
          getField(value); return value; }
        public bool isSet(QuickFix.PartySubID field)
        { return isSetField(field); }
        public bool isSetPartySubID()
        { return isSetField(523); }

        public void set(QuickFix.PartySubIDType value)
        { setField(value); }
        public QuickFix.PartySubIDType get(QuickFix.PartySubIDType  value)
        { getField(value); return value; }
        public QuickFix.PartySubIDType getPartySubIDType()
        { QuickFix.PartySubIDType value = new QuickFix.PartySubIDType();
          getField(value); return value; }
        public bool isSet(QuickFix.PartySubIDType field)
        { return isSetField(field); }
        public bool isSetPartySubIDType()
        { return isSetField(803); }

      };
    };
    public void set(QuickFix.AllocAccount value)
    { setField(value); }
    public QuickFix.AllocAccount get(QuickFix.AllocAccount  value)
    { getField(value); return value; }
    public QuickFix.AllocAccount getAllocAccount()
    { QuickFix.AllocAccount value = new QuickFix.AllocAccount();
      getField(value); return value; }
    public bool isSet(QuickFix.AllocAccount field)
    { return isSetField(field); }
    public bool isSetAllocAccount()
    { return isSetField(79); }

    public void set(QuickFix.AllocAcctIDSource value)
    { setField(value); }
    public QuickFix.AllocAcctIDSource get(QuickFix.AllocAcctIDSource  value)
    { getField(value); return value; }
    public QuickFix.AllocAcctIDSource getAllocAcctIDSource()
    { QuickFix.AllocAcctIDSource value = new QuickFix.AllocAcctIDSource();
      getField(value); return value; }
    public bool isSet(QuickFix.AllocAcctIDSource field)
    { return isSetField(field); }
    public bool isSetAllocAcctIDSource()
    { return isSetField(661); }

    public void set(QuickFix.Side value)
    { setField(value); }
    public QuickFix.Side get(QuickFix.Side  value)
    { getField(value); return value; }
    public QuickFix.Side getSide()
    { QuickFix.Side value = new QuickFix.Side();
      getField(value); return value; }
    public bool isSet(QuickFix.Side field)
    { return isSetField(field); }
    public bool isSetSide()
    { return isSetField(54); }

    public void set(QuickFix.Product value)
    { setField(value); }
    public QuickFix.Product get(QuickFix.Product  value)
    { getField(value); return value; }
    public QuickFix.Product getProduct()
    { QuickFix.Product value = new QuickFix.Product();
      getField(value); return value; }
    public bool isSet(QuickFix.Product field)
    { return isSetField(field); }
    public bool isSetProduct()
    { return isSetField(460); }

    public void set(QuickFix.SecurityType value)
    { setField(value); }
    public QuickFix.SecurityType get(QuickFix.SecurityType  value)
    { getField(value); return value; }
    public QuickFix.SecurityType getSecurityType()
    { QuickFix.SecurityType value = new QuickFix.SecurityType();
      getField(value); return value; }
    public bool isSet(QuickFix.SecurityType field)
    { return isSetField(field); }
    public bool isSetSecurityType()
    { return isSetField(167); }

    public void set(QuickFix.CFICode value)
    { setField(value); }
    public QuickFix.CFICode get(QuickFix.CFICode  value)
    { getField(value); return value; }
    public QuickFix.CFICode getCFICode()
    { QuickFix.CFICode value = new QuickFix.CFICode();
      getField(value); return value; }
    public bool isSet(QuickFix.CFICode field)
    { return isSetField(field); }
    public bool isSetCFICode()
    { return isSetField(461); }

    public void set(QuickFix.EffectiveTime value)
    { setField(value); }
    public QuickFix.EffectiveTime get(QuickFix.EffectiveTime  value)
    { getField(value); return value; }
    public QuickFix.EffectiveTime getEffectiveTime()
    { QuickFix.EffectiveTime value = new QuickFix.EffectiveTime();
      getField(value); return value; }
    public bool isSet(QuickFix.EffectiveTime field)
    { return isSetField(field); }
    public bool isSetEffectiveTime()
    { return isSetField(168); }

    public void set(QuickFix.ExpireTime value)
    { setField(value); }
    public QuickFix.ExpireTime get(QuickFix.ExpireTime  value)
    { getField(value); return value; }
    public QuickFix.ExpireTime getExpireTime()
    { QuickFix.ExpireTime value = new QuickFix.ExpireTime();
      getField(value); return value; }
    public bool isSet(QuickFix.ExpireTime field)
    { return isSetField(field); }
    public bool isSetExpireTime()
    { return isSetField(126); }

    public void set(QuickFix.LastUpdateTime value)
    { setField(value); }
    public QuickFix.LastUpdateTime get(QuickFix.LastUpdateTime  value)
    { getField(value); return value; }
    public QuickFix.LastUpdateTime getLastUpdateTime()
    { QuickFix.LastUpdateTime value = new QuickFix.LastUpdateTime();
      getField(value); return value; }
    public bool isSet(QuickFix.LastUpdateTime field)
    { return isSetField(field); }
    public bool isSetLastUpdateTime()
    { return isSetField(779); }

    public void set(QuickFix.StandInstDbType value)
    { setField(value); }
    public QuickFix.StandInstDbType get(QuickFix.StandInstDbType  value)
    { getField(value); return value; }
    public QuickFix.StandInstDbType getStandInstDbType()
    { QuickFix.StandInstDbType value = new QuickFix.StandInstDbType();
      getField(value); return value; }
    public bool isSet(QuickFix.StandInstDbType field)
    { return isSetField(field); }
    public bool isSetStandInstDbType()
    { return isSetField(169); }

    public void set(QuickFix.StandInstDbName value)
    { setField(value); }
    public QuickFix.StandInstDbName get(QuickFix.StandInstDbName  value)
    { getField(value); return value; }
    public QuickFix.StandInstDbName getStandInstDbName()
    { QuickFix.StandInstDbName value = new QuickFix.StandInstDbName();
      getField(value); return value; }
    public bool isSet(QuickFix.StandInstDbName field)
    { return isSetField(field); }
    public bool isSetStandInstDbName()
    { return isSetField(170); }

    public void set(QuickFix.StandInstDbID value)
    { setField(value); }
    public QuickFix.StandInstDbID get(QuickFix.StandInstDbID  value)
    { getField(value); return value; }
    public QuickFix.StandInstDbID getStandInstDbID()
    { QuickFix.StandInstDbID value = new QuickFix.StandInstDbID();
      getField(value); return value; }
    public bool isSet(QuickFix.StandInstDbID field)
    { return isSetField(field); }
    public bool isSetStandInstDbID()
    { return isSetField(171); }

  };

}

File: src/java/src/org/quickfix/fix44/SettlementInstructionRequest.java
package org.quickfix.fix44; 
import org.quickfix.FieldNotFound; 
import org.quickfix.Group; 
import org.quickfix.field.*; 

public class SettlementInstructionRequest extends Message 
{ 

  public SettlementInstructionRequest() 
  { 
    getHeader().setField(new MsgType("AV")); 
  } 
  public SettlementInstructionRequest(    
    org.quickfix.field.SettlInstReqID aSettlInstReqID,    
    org.quickfix.field.TransactTime aTransactTime ) 
  {  
    getHeader().setField(new MsgType("AV")); 
    set(aSettlInstReqID); 
    set(aTransactTime);  
  } 

  public void set(org.quickfix.field.SettlInstReqID value) 
  { setField(value); } 
  public org.quickfix.field.SettlInstReqID get(org.quickfix.field.SettlInstReqID value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.SettlInstReqID getSettlInstReqID() throws FieldNotFound 
  { org.quickfix.field.SettlInstReqID value = new org.quickfix.field.SettlInstReqID(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.SettlInstReqID field) 
  { return isSetField(field); } 
  public boolean isSetSettlInstReqID() 
  { return isSetField(791); } 

  public void set(org.quickfix.field.TransactTime value) 
  { setField(value); } 
  public org.quickfix.field.TransactTime get(org.quickfix.field.TransactTime value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.TransactTime getTransactTime() throws FieldNotFound 
  { org.quickfix.field.TransactTime value = new org.quickfix.field.TransactTime(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.TransactTime field) 
  { return isSetField(field); } 
  public boolean isSetTransactTime() 
  { return isSetField(60); } 
  public void set(org.quickfix.field.NoPartyIDs value) 
  { setField(value); } 
  public org.quickfix.field.NoPartyIDs get(org.quickfix.field.NoPartyIDs value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.NoPartyIDs getNoPartyIDs() throws FieldNotFound 
  { org.quickfix.field.NoPartyIDs value = new org.quickfix.field.NoPartyIDs(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.NoPartyIDs field) 
  { return isSetField(field); } 
  public boolean isSetNoPartyIDs() 
  { return isSetField(453); } 

public static class NoPartyIDs extends Group { 
  public NoPartyIDs() { 
    super(453, 447, 
    new int[] { 
               448, 
               447, 
               452, 
               0 } 
); 
  } 
  public void set(org.quickfix.field.PartyID value) 
  { setField(value); } 
  public org.quickfix.field.PartyID get(org.quickfix.field.PartyID value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.PartyID getPartyID() throws FieldNotFound 
  { org.quickfix.field.PartyID value = new org.quickfix.field.PartyID(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.PartyID field) 
  { return isSetField(field); } 
  public boolean isSetPartyID() 
  { return isSetField(448); } 

  public void set(org.quickfix.field.PartyIDSource value) 
  { setField(value); } 
  public org.quickfix.field.PartyIDSource get(org.quickfix.field.PartyIDSource value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.PartyIDSource getPartyIDSource() throws FieldNotFound 
  { org.quickfix.field.PartyIDSource value = new org.quickfix.field.PartyIDSource(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.PartyIDSource field) 
  { return isSetField(field); } 
  public boolean isSetPartyIDSource() 
  { return isSetField(447); } 

  public void set(org.quickfix.field.PartyRole value) 
  { setField(value); } 
  public org.quickfix.field.PartyRole get(org.quickfix.field.PartyRole value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.PartyRole getPartyRole() throws FieldNotFound 
  { org.quickfix.field.PartyRole value = new org.quickfix.field.PartyRole(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.PartyRole field) 
  { return isSetField(field); } 
  public boolean isSetPartyRole() 
  { return isSetField(452); } 

  public void set(org.quickfix.field.NoPartySubIDs value) 
  { setField(value); } 
  public org.quickfix.field.NoPartySubIDs get(org.quickfix.field.NoPartySubIDs value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.NoPartySubIDs getNoPartySubIDs() throws FieldNotFound 
  { org.quickfix.field.NoPartySubIDs value = new org.quickfix.field.NoPartySubIDs(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.NoPartySubIDs field) 
  { return isSetField(field); } 
  public boolean isSetNoPartySubIDs() 
  { return isSetField(802); } 

public static class NoPartySubIDs extends Group { 
  public NoPartySubIDs() { 
    super(802, 523, 
    new int[] { 
               523, 
               803, 
               0 } 
); 
  } 
  public void set(org.quickfix.field.PartySubID value) 
  { setField(value); } 
  public org.quickfix.field.PartySubID get(org.quickfix.field.PartySubID value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.PartySubID getPartySubID() throws FieldNotFound 
  { org.quickfix.field.PartySubID value = new org.quickfix.field.PartySubID(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.PartySubID field) 
  { return isSetField(field); } 
  public boolean isSetPartySubID() 
  { return isSetField(523); } 

  public void set(org.quickfix.field.PartySubIDType value) 
  { setField(value); } 
  public org.quickfix.field.PartySubIDType get(org.quickfix.field.PartySubIDType value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.PartySubIDType getPartySubIDType() throws FieldNotFound 
  { org.quickfix.field.PartySubIDType value = new org.quickfix.field.PartySubIDType(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.PartySubIDType field) 
  { return isSetField(field); } 
  public boolean isSetPartySubIDType() 
  { return isSetField(803); } 

} 
} 

  public void set(org.quickfix.field.AllocAccount value) 
  { setField(value); } 
  public org.quickfix.field.AllocAccount get(org.quickfix.field.AllocAccount value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.AllocAccount getAllocAccount() throws FieldNotFound 
  { org.quickfix.field.AllocAccount value = new org.quickfix.field.AllocAccount(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.AllocAccount field) 
  { return isSetField(field); } 
  public boolean isSetAllocAccount() 
  { return isSetField(79); } 

  public void set(org.quickfix.field.AllocAcctIDSource value) 
  { setField(value); } 
  public org.quickfix.field.AllocAcctIDSource get(org.quickfix.field.AllocAcctIDSource value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.AllocAcctIDSource getAllocAcctIDSource() throws FieldNotFound 
  { org.quickfix.field.AllocAcctIDSource value = new org.quickfix.field.AllocAcctIDSource(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.AllocAcctIDSource field) 
  { return isSetField(field); } 
  public boolean isSetAllocAcctIDSource() 
  { return isSetField(661); } 

  public void set(org.quickfix.field.Side value) 
  { setField(value); } 
  public org.quickfix.field.Side get(org.quickfix.field.Side value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.Side getSide() throws FieldNotFound 
  { org.quickfix.field.Side value = new org.quickfix.field.Side(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.Side field) 
  { return isSetField(field); } 
  public boolean isSetSide() 
  { return isSetField(54); } 

  public void set(org.quickfix.field.Product value) 
  { setField(value); } 
  public org.quickfix.field.Product get(org.quickfix.field.Product value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.Product getProduct() throws FieldNotFound 
  { org.quickfix.field.Product value = new org.quickfix.field.Product(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.Product field) 
  { return isSetField(field); } 
  public boolean isSetProduct() 
  { return isSetField(460); } 

  public void set(org.quickfix.field.SecurityType value) 
  { setField(value); } 
  public org.quickfix.field.SecurityType get(org.quickfix.field.SecurityType value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.SecurityType getSecurityType() throws FieldNotFound 
  { org.quickfix.field.SecurityType value = new org.quickfix.field.SecurityType(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.SecurityType field) 
  { return isSetField(field); } 
  public boolean isSetSecurityType() 
  { return isSetField(167); } 

  public void set(org.quickfix.field.CFICode value) 
  { setField(value); } 
  public org.quickfix.field.CFICode get(org.quickfix.field.CFICode value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.CFICode getCFICode() throws FieldNotFound 
  { org.quickfix.field.CFICode value = new org.quickfix.field.CFICode(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.CFICode field) 
  { return isSetField(field); } 
  public boolean isSetCFICode() 
  { return isSetField(461); } 

  public void set(org.quickfix.field.EffectiveTime value) 
  { setField(value); } 
  public org.quickfix.field.EffectiveTime get(org.quickfix.field.EffectiveTime value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.EffectiveTime getEffectiveTime() throws FieldNotFound 
  { org.quickfix.field.EffectiveTime value = new org.quickfix.field.EffectiveTime(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.EffectiveTime field) 
  { return isSetField(field); } 
  public boolean isSetEffectiveTime() 
  { return isSetField(168); } 

  public void set(org.quickfix.field.ExpireTime value) 
  { setField(value); } 
  public org.quickfix.field.ExpireTime get(org.quickfix.field.ExpireTime value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.ExpireTime getExpireTime() throws FieldNotFound 
  { org.quickfix.field.ExpireTime value = new org.quickfix.field.ExpireTime(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.ExpireTime field) 
  { return isSetField(field); } 
  public boolean isSetExpireTime() 
  { return isSetField(126); } 

  public void set(org.quickfix.field.LastUpdateTime value) 
  { setField(value); } 
  public org.quickfix.field.LastUpdateTime get(org.quickfix.field.LastUpdateTime value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.LastUpdateTime getLastUpdateTime() throws FieldNotFound 
  { org.quickfix.field.LastUpdateTime value = new org.quickfix.field.LastUpdateTime(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.LastUpdateTime field) 
  { return isSetField(field); } 
  public boolean isSetLastUpdateTime() 
  { return isSetField(779); } 

  public void set(org.quickfix.field.StandInstDbType value) 
  { setField(value); } 
  public org.quickfix.field.StandInstDbType get(org.quickfix.field.StandInstDbType value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.StandInstDbType getStandInstDbType() throws FieldNotFound 
  { org.quickfix.field.StandInstDbType value = new org.quickfix.field.StandInstDbType(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.StandInstDbType field) 
  { return isSetField(field); } 
  public boolean isSetStandInstDbType() 
  { return isSetField(169); } 

  public void set(org.quickfix.field.StandInstDbName value) 
  { setField(value); } 
  public org.quickfix.field.StandInstDbName get(org.quickfix.field.StandInstDbName value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.StandInstDbName getStandInstDbName() throws FieldNotFound 
  { org.quickfix.field.StandInstDbName value = new org.quickfix.field.StandInstDbName(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.StandInstDbName field) 
  { return isSetField(field); } 
  public boolean isSetStandInstDbName() 
  { return isSetField(170); } 

  public void set(org.quickfix.field.StandInstDbID value) 
  { setField(value); } 
  public org.quickfix.field.StandInstDbID get(org.quickfix.field.StandInstDbID value) throws FieldNotFound 
  { getField(value); return value; } 
  public org.quickfix.field.StandInstDbID getStandInstDbID() throws FieldNotFound 
  { org.quickfix.field.StandInstDbID value = new org.quickfix.field.StandInstDbID(); 
    getField(value); return value; } 
  public boolean isSet(org.quickfix.field.StandInstDbID field) 
  { return isSetField(field); } 
  public boolean isSetStandInstDbID() 
  { return isSetField(171); } 
} 
